docs/multi-plant_tutorial.md

```diff
@@ -350,13 +350,36 @@ for an example.
     clusters_i, contours, hierarchies = pcv.cluster_contours(img=img1, roi_objects=roi_objects, 
                                                              roi_obj_hierarchy=roi_obj_hierarchy, 
                                                              nrow=4, ncol=6)
-
 ```
 
-**Figure 12.** Cluster contours
+**Figure 12a.** Cluster contours
 
 ![Screenshot](img/tutorial_images/multi-img/12_clusters.jpg)
 
+```python
+
+    # Step 12b: This function is a plotting method used to represent 
+    # the way in which contours get clustered together into objects
+    
+    # Inputs:
+    #   img                      = RGB or grayscale image data for plotting.
+    #   grouped_contour_indices  = Contour indices for which groups contours belong to. Output from [pcv.cluster_contour](cluster_contours.md)
+    #   roi_objects              = object contours in an image that are needed to be clustered.
+    #   roi_obj_hierarchy        = object hierarchy
+    #   nrow                     = Optional row grid lines to get drawn (default nrow=1). If `nrow` is unchanged, grid lines don't get drawn. 
+    #   ncol                     = Optional column grid lines to get drawn (default ncol=1). If `ncol` is unchanged, grid lines don't get drawn. 
+    #   bounding                 = Optional circles to bound the individual clusters (default bounding=True)
+
+    clustered_image = pcv.visualize.clustered_contours(img=img, grouped_contour_indices=cnt_i, 
+                                                   roi_objects=roi_obj,
+                                                   roi_obj_hierarchy=hier,
+                                                   nrow=4, ncol=6)
+```
```

Comment:
Add "bounding=True" inside the example of the function getting called. It's true that this is redundant due to it's default, but to make things abundantly clear and consistent for tutorials :)